ext/drx/drx.c

```diff
@@ -219,32 +219,10 @@ DR_EXPORT
 bool
 drx_aflags_are_dead(instr_t *where)
 {
-    instr_t *instr;
-    uint flags;
-    for (instr = where; instr != NULL; instr = instr_get_next(instr)) {
-        /* we treat syscall/interrupt as aflags read */
-        if (instr_is_syscall(instr) || instr_is_interrupt(instr))
-            return false;
-        flags = instr_get_arith_flags(instr, DR_QUERY_DEFAULT);
-        if (TESTANY(EFLAGS_READ_ARITH, flags))
-            return false;
-        if (TESTALL(EFLAGS_WRITE_ARITH, flags))
-            return true;
-        if (instr_is_cti(instr)) {
-            if (instr_is_app(instr) &&
-                (instr_is_ubr(instr) || instr_is_call_direct(instr))) {
-                instr_t *next = instr_get_next(instr);
-                opnd_t tgt = instr_get_target(instr);
-                /* continue on elision */
-                if (next != NULL && instr_is_app(next) && opnd_is_pc(tgt) &&
-                    opnd_get_pc(tgt) == instr_get_app_pc(next))
-                    continue;
-            }
-            /* unknown target, assume aflags is live */
-            return false;
-        }
-    }
-    return false;
+    bool dead = false;
+    drreg_status_t res = drreg_are_aflags_dead(dr_get_current_drcontext(), where, &dead);
```

Comment:
You can use `IF_DEBUG(res =)` to fix the unused var error, like elsewhere in drx.c too.

Reply:
Thanks! I replace DR_ASSERT_MSG with IF_DEBUG in the latest commit.